[0m15:41:22.354967 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7422c3568c50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7422c17354f0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7422c30a56d0>]}


============================== 15:41:22.368067 | 717d8f67-8597-4333-9f6e-27f95b51ba64 ==============================
[0m15:41:22.368067 [info ] [MainThread]: Running with dbt=1.9.0
[0m15:41:22.369253 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'version_check': 'True', 'profiles_dir': '/root/.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'invocation_command': 'dbt init my_project', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m15:41:22.414441 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse logging event dictionary. Failed to parse dir field: expected string or bytes-like object, got 'PosixPath'.. Dictionary: {'dir': PosixPath('/root/.dbt')}
[0m15:41:22.415049 [info ] [MainThread]: Creating dbt configuration folder at 
[0m15:41:22.419365 [debug] [MainThread]: Starter project path: /usr/local/lib/python3.12/site-packages/dbt/include/starter_project
[0m15:41:22.423623 [info ] [MainThread]: 
Your new dbt project "my_project" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m15:41:22.424051 [info ] [MainThread]: Setting up your profile.
[0m15:41:22.424887 [error] [MainThread]: Encountered an error:

[0m15:41:22.427752 [error] [MainThread]: Traceback (most recent call last):
  File "/usr/local/lib/python3.12/site-packages/dbt/cli/requires.py", line 153, in wrapper
    result, success = func(*args, **kwargs)
                      ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/dbt/cli/requires.py", line 103, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/dbt/cli/main.py", line 477, in init
    results = task.run()
              ^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/dbt/task/init.py", line 346, in run
    self.setup_profile(profile_name)
  File "/usr/local/lib/python3.12/site-packages/dbt/task/init.py", line 261, in setup_profile
    adapter = self.ask_for_adapter_choice()
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/dbt/task/init.py", line 242, in ask_for_adapter_choice
    numeric_choice = click.prompt(prompt_msg, type=click.INT)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/click/termui.py", line 164, in prompt
    value = prompt_func(prompt)
            ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/click/termui.py", line 147, in prompt_func
    raise Abort() from None
click.exceptions.Abort

[0m15:41:22.429951 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": false, "command_wall_clock_time": 0.2652675, "process_in_blocks": "3488", "process_kernel_time": 0.256343, "process_mem_max_rss": "87392", "process_out_blocks": "1528", "process_user_time": 1.474218}
[0m15:41:22.430630 [debug] [MainThread]: Command `dbt init` failed at 15:41:22.430470 after 0.27 seconds
[0m15:41:22.432143 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7422c17f3380>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7422c141ccb0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7422c1451d60>]}
[0m15:41:22.432938 [debug] [MainThread]: Flushing usage events
[0m15:41:23.609801 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m15:46:49.641725 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7c53c211c290>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7c53c284c9b0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7c53c2734e90>]}


============================== 15:46:49.648764 | ba06e409-5aea-4772-aeb5-d74a23559026 ==============================
[0m15:46:49.648764 [info ] [MainThread]: Running with dbt=1.9.0
[0m15:46:49.649503 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/root/.dbt', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'debug': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'invocation_command': 'dbt debug', 'static_parser': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m15:46:49.690073 [info ] [MainThread]: dbt version: 1.9.0
[0m15:46:49.690524 [info ] [MainThread]: python version: 3.12.4
[0m15:46:49.690948 [info ] [MainThread]: python path: /usr/local/bin/python
[0m15:46:49.691364 [info ] [MainThread]: os info: Linux-6.8.0-1030-azure-x86_64-with-glibc2.31
[0m15:46:49.691843 [info ] [MainThread]: Using profiles dir at /root/.dbt
[0m15:46:49.692264 [info ] [MainThread]: Using profiles.yml file at /root/.dbt/profiles.yml
[0m15:46:49.692725 [info ] [MainThread]: Using dbt_project.yml file at /user/app/dbt_project.yml
[0m15:46:49.693174 [info ] [MainThread]: Configuration:
[0m15:46:49.693602 [info ] [MainThread]:   profiles.yml file [[31mERROR not found[0m]
[0m15:46:49.694057 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m15:46:49.694471 [info ] [MainThread]: Required dependencies:
[0m15:46:49.694951 [debug] [MainThread]: Executing "git --help"
[0m15:46:49.708393 [debug] [MainThread]: STDOUT: "b"usage: git [--version] [--help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]\n           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]\n           <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone             Clone a repository into a new directory\n   init              Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add               Add file contents to the index\n   mv                Move or rename a file, a directory, or a symlink\n   restore           Restore working tree files\n   rm                Remove files from the working tree and from the index\n   sparse-checkout   Initialize and modify the sparse-checkout\n\nexamine the history and state (see also: git help revisions)\n   bisect            Use binary search to find the commit that introduced a bug\n   diff              Show changes between commits, commit and working tree, etc\n   grep              Print lines matching a pattern\n   log               Show commit logs\n   show              Show various types of objects\n   status            Show the working tree status\n\ngrow, mark and tweak your common history\n   branch            List, create, or delete branches\n   commit            Record changes to the repository\n   merge             Join two or more development histories together\n   rebase            Reapply commits on top of another base tip\n   reset             Reset current HEAD to the specified state\n   switch            Switch branches\n   tag               Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch             Download objects and refs from another repository\n   pull              Fetch from and integrate with another repository or a local branch\n   push              Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m15:46:49.709087 [debug] [MainThread]: STDERR: "b''"
[0m15:46:49.709503 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m15:46:49.709976 [info ] [MainThread]: Connection test skipped since no profile was found
[0m15:46:49.710478 [info ] [MainThread]: [31m2 checks failed:[0m
[0m15:46:49.711778 [info ] [MainThread]: dbt looked for a profiles.yml file in /root/.dbt/profiles.yml, but did
not find one. For more information on configuring your profile, consult the
documentation:

https://docs.getdbt.com/docs/configure-your-profile


[0m15:46:49.712224 [info ] [MainThread]: Project loading failed for the following reason:
 project path </user/app/dbt_project.yml> not found

[0m15:46:49.713509 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_success": false, "command_wall_clock_time": 0.12262242, "process_in_blocks": "0", "process_kernel_time": 0.209772, "process_mem_max_rss": "87324", "process_out_blocks": "1480", "process_user_time": 1.364518}
[0m15:46:49.715236 [debug] [MainThread]: Command `dbt debug` failed at 15:46:49.714740 after 0.12 seconds
[0m15:46:49.716101 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7c53c2482db0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7c53c1aa0410>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7c53c3b34650>]}
[0m15:46:49.716885 [debug] [MainThread]: Flushing usage events
[0m15:46:50.883391 [debug] [MainThread]: An error was encountered while trying to flush usage events
